PMMain/Midas/MidasGUI/DistanceBar.h

```diff
@@ -0,0 +1,60 @@
+#ifndef DISTANCE_BAR_H
+#define DISTANCE_BAR_H
+
+#include <MidasCommon.h>
+#include <QGraphicsRectItem>
+#include <qlabel.h>
+#include <qpainter.h>
+#include <QPaintEvent>
+#include <qrect.h>
+#include <qwidget.h>
+
+class distanceBar : public QWidget
+{
+    Q_OBJECT
+public:
+    distanceBar(qreal x, qreal y, QWidget *parent = 0, qreal width = DISTANCE_DISPLAY_WIDTH,
+        qreal height = DISTANCE_DISPLAY_HEIGHT);
+ 
+    enum class bins{
+        SIG_OOR = 0,
+        SIG_FAR = 1,
+        SIG_NEAR = 2,
+        SIG_IMMEDIATE = 3
+    };
+
+public slots:
+    void setValue(int bin);
+
+protected:
+    void paintEvent(QPaintEvent *e);
+        
+private:
+    QWidget *parent;
+
+    QRect *long_bar;
+    QGraphicsRectItem *med_bar;
+    QGraphicsRectItem *short_bar;
```

Comment:
It was supposed to be for the future work where there were three bars of different sizes. med_bar and short_bar are not used. I will remove them.